src/poc/miner_poc_statem.erl

```diff
@@ -70,6 +70,7 @@
     responses = #{},
     receiving_timeout = ?RECEIVING_TIMEOUT :: non_neg_integer(),
     poc_hash  :: binary() | undefined,
+    request_block_hash  :: binary() | undefined,
```

Comment:
default this to `<<>>`?  otherwise we'll get a lot of crashes on the transition.  might never see it, though.

Reply:
I'm not sure how that will help?